README.md

```diff
@@ -31,24 +29,21 @@ Twilio Passkeys SDK enables developers to easily add Passkeys into their existin
 - Gradle 8.2
 - Java 17
 
-## Installation <a name="installation"></a>
-
-### Installation Android
-1. Download the .aar file from the [release page](https://github.com/twilio/twilio-verify-passkeys/releases).
-2. Create a folder `libs` in the module directory.
-3. Copy/move the .aar file in `libs` folder.
-4. Add the implementation statement in dependencies:
-```
-implementation(files("libs/TwilioPasskeys.aar"))
-```
-5. Sync the project.
-6. Use the SDK by creating an instance of TwilioPasskey:
-```
-val twilioPasskey = TwilioPasskey(context)
-```
+## Quickstart <a name="quickstart"></a>
 
-### Installation iOS
+### Initialize the SDK
+#### Android
+1. Add the implementation app build gradle:
+  ```
+    implementation("com.twilio:twilio-verify-passkeys-android:$sdkVersionName")
+  ```
+2. Sync the project.
+3. Use the SDK by creating an instance of TwilioPasskey:
+  ```
+    val twilioPasskey = TwilioPasskey(context)
+  ```
 
+#### iOS
 1. Download the XCFramework form the [release page](https://github.com/twilio/twilio-verify-passkeys/releases).
```

Comment:
Is this still true? @AlejandroOrozco Is there not SPM available?

Reply:
We do have SPM available, and I think we should redirect to the specific iOS repo where the binary is being published. https://github.com/twilio/twilio-verify-passkeys-ios/releases

Reply:
@SergioFierro could you please include this change

Reply:
Done